.github/workflows/deep-inspect-multi-arch.yaml

```
@@ -562,35 +562,12 @@ jobs:
-              break
-            elif [ $attempt -eq 5 ]; then
-              echo "::error::Failed to push after 5 attempts due to concurrent build conflicts."
-              exit 1
-            fi
-            echo "::warning::Push attempt $attempt/5 failed (remote is ahead), rebasing and retrying..."
-            git checkout -- .
-            git clean -fd
-            git pull --rebase
-            sleep $((RANDOM % 10 + 5))
-          done
+        uses: ./.github/actions/publish-with-retry
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/publish-with-retry": yaml: line 25: could not find expected ':'

(action)